.rubocop.yml

```diff
@@ -0,0 +1,10 @@
+require: rubocop-rspec
```

Comment:
My advice is to use more strict rules for the Rubocop. This is one of the easiest tricks helping you write the clean code. Moreover, it doesn't really matter which rules you have. The point in having it.

Take a look at the [config](https://gist.github.com/atatarenko/c72a6e01cde7a96ee10527f49a91bbc5) from the current project. It isn't perfect, really, but more strict.